### PR TITLE
style create password page

### PR DESCRIPTION
**Pull request by TraceyOnim**

### CODE REVIEW

File: assets/package.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "license": "Apache-2.0",
   "devDependencies": {
+    "@tailwindcss/custom-forms": "^0.2.1",
```
Comment (ericmj) [inline assets/package.json:4]: This package is deprecated in favor of `@tailwindcss/forms`. See https://github.com/tailwindlabs/tailwindcss-custom-forms#readme.

File: assets/tailwind.config.js [hunk 2]
```diff
@@ -91,7 +91,9 @@ module.exports = {
   variants: {
     extend: {}
   },
-  plugins: []
+  plugins: [
+  require('@tailwindcss/custom-forms')
```
[the remark this answers is not in the document]
Comment (ericmj) [reply assets/tailwind.config.js:95]:
```suggestion
    require('@tailwindcss/custom-forms')
```

File: assets/package.json (hunk 1 above)
Comment (TraceyOnim) [reply assets/package.json:4]: Thank you, working on it

**Review by ericmj — COMMENTED** [review]
This looks good, just two small comments.

The spacing between the validation error and the checkbox is too small when there is an error:
Screenshot 2021-12-02 at 18 00 37

We also need to add a red border and an exclamation point icon when there is a validation error but that can be done in a separate PR.

**Comment by TraceyOnim** [comment]
> This looks good, just two small comments.
> 
> The spacing between the validation error and the checkbox is too small when there is an error: Screenshot 2021-12-02 at 18 00 37
> Lemme adjust that
> We also need to add a red border and an exclamation point icon when there is a validation error but that can be done in a separate PR.
  I will add this as my next task

File: assets/package.json [hunk 3]
```diff
@@ -8,5 +8,8 @@
     "postcss-import": "^14.0.2",
     "postcss-loader": "^4.2.0",
     "tailwindcss": "^2.2.19"
+  },
+  "dependencies": {
+    "@tailwindcss/forms": "^0.3.4"
```
[the remark this answers is not in the document]
Comment (TraceyOnim) [reply assets/package.json:13]: Yeah it should, though on the documentation(https://github.com/tailwindlabs/tailwindcss-forms) they had not specified so I wasn't quite sure where to add it .

**Pull request merged**